js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
Check out the lines below; this is what my User Prefs look like in Sublime. Go to the menu selection: Sublime Text > Preferences > Settings-User.

If you’ve not set anything it will be blank. You can insert the code below in part or in whole to add these settings to your prefs. You will want at least the two lines for “tab_size” and “translate_tabs_to_spaces”. 

If you’ve already done that, let me know and we can take a closer look!

{
"ensure_newline_at_eof_on_save": true,
"font_size": 14.0,
"ignored_packages":
[
"Vintage",
],
"save_on_focus_lost": true,
"scroll_past_end": true,
"tab_size": 2,
"translate_tabs_to_spaces": true,
"word_wrap": false
}


From:  Isabella Organ
Reply-To:  isabellaorgan/vote_tracker
Date:  Thursday, September 3, 2015 at 3:17 PM
To:  isabellaorgan/vote_tracker
Cc:  Scott Schmidt
Subject:  Re: [vote_tracker] photos click, work on tally (#4)


In js/app.js:
> @@ -13,6 +13,8 @@ var Tracker = function () {
>  	this.photoArray = [];
>  }
>  
> +var tracker = new Tracker();
I have been trying to convince Sublime to allow me to change these settings for weeks now. I also get an error message when I open Sublime that Sam and I had intended to work on fixing, but didn't get around to it. I guess I have another project to add to my list!

—
Reply to this email directly or view it on GitHub.